sys-devel/bmake/bmake-20210704.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MK_VER=20210330
```

Comment:
There seems to be a later version available... but I don't know anything if it's compatible or not. Just FYI.

Reply:
We don't update MK_VER on every version bump, since it doesn't make much difference on which version we pull in I think it's okay like this